XenAdmin/Controls/TreeViews/FlickerFreeTreeView.cs

```diff
@@ -157,7 +157,7 @@ public new void EndUpdate()
             {
                 VirtualTreeNode.PersistenceInfo info = new VirtualTreeNode.PersistenceInfo(node);
 
-                if (!_persistedSelectionInfo.Contains(info))
+                if (_persistedSelectionInfo != null && !_persistedSelectionInfo.Contains(info))

```

Comment:
Actually can't because the `?.` operation doesn't return `bool`. This doesn't compile:
![image](https://user-images.githubusercontent.com/32554698/140040322-a915ba38-a4d8-40be-b1a2-33a9b34130c0.png)